doc/ref_impl/cntt-ri/chapters/chapter06.md

```diff
@@ -61,241 +63,400 @@ Thanks to the descriptor file, the NFVi infrastructure deployment could be compl
+| VIM_CERTIFICATES_PATH | String | Yes | Full path for certificates that used for integration |
 
-<p align="center"><b>Table 6-3-12:</b> Auto deployment control information.</p>
+<p align="center"><b>Table 6-3-14-1:</b> VIM context Information.</p>
 
-<a name="6.3.13"></a>
-### 6.3.13 Proxy information
-Proxy information, this section could be empty if not needed.
+#### 6.3.14.2 Storage Context
+Parameters from storage vendor for integration.
+
+Only one entry is expected.
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| VENDOR | String | Yes |  |
+| AUTHORIZATION | String | Yes | One-way or Two-way authentication |
+| JOINT_WAY | String | Yes | by ISCSI or client |
+| DRIVER_FULL_NAME | String | Yes | Full path for storage driver |
+| CEPH_CONFIG_PATH | String | Yes | Full path for ceph.conf storage |
+| IS_PIM_JOINT | String | Yes | whether integrate with PIM, usaually "YES" for this value |
+| STORAGE_SOFTWARE_VERSION | String | Yes |  |
+
+<p align="center"><b>Table 6-3-14-2:</b> Storage context Information.</p>
 
+#### 6.3.14.3 Storage Client context
+This table defines the parameters for integration with storage client
+
+Multiple entries are expected, one entry for each authorization user.
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| JOINT_WAY | String | Yes | integration method for storage client, for example, RBD  |
+| COMPONENT_TYPE | String | Yes | for example: cinder, glance or nova |
+| AUTHORIZATION_USER | String | Yes | match with the component type |
+| KEYRING_FILENAME | String | Yes | Full path for keyring file, this should match the authentication user, |
+
+<p align="center"><b>Table 6-3-14-3:</b> Storage client context.</p>
+
+<a name="6.3.15"></a>
+### 6.3.15 Device Management information
+
+#### 6.3.15.1 SERVER PIM ACCOUNT
+Servers are managed by redfish, credentials should be the same for same type of device  
+
+Multiple entries are expected, one entry for each server model.
 | Field # | type | mandatory | Instruction |
 |----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| proxyAddress | String | Yes | Proxy address |
-| port | String | Yes | port |
-| user | String | Yes | user |
-| password | String | Yes | password |
+| VENDOR | String | Yes |  |
+| SERVER_MODEL | String | Yes | MODEL for each type of server |
+| REDFISH_USER | String | Yes |  |
```

Comment:
Mandatory = yes. Only servers that support Redfish?

Reply:
I think so, only server support Redfish
and all servers should support Redfish.

Am I wrong?

Reply:
I don't think that any of the servers implementing RI1 currently support Redfish. As I understand its only Airship that requires Redfish. Other installers may not require it and all vendors do not support it?

Reply:
In our internal projects of building up NFVi, installers do support redfish, and CIRV-HDV redfish related testcases.
Maybe it's not the case for others especially internationally.

I have update this redfish to not mandatory, as long as servers able to be managed by Physical Infrastructure Manager, it would be good.